### PR TITLE
Add the policy addon overview video link

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

File: content/en/blog/_index.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ title: Blog
 weight: 6
 ---
 
+---
```
Comment (dhaiducek) [inline content/en/blog/_index.md:6]: Optional nit: While technically correct, this uses 3 dashes while the others use four
```suggestion
----
```

**Review by dhaiducek — APPROVED** [review]
/hold

Thanks, @mprahl! Optional nit to consider 🙂

File: content/en/blog/_index.md (hunk 1 above)
Comment (mprahl) [reply content/en/blog/_index.md]: I fixed it so you can rest easy tonight.  :wink:

Comment (dhaiducek) [reply content/en/blog/_index.md]: Haha--thanks! You know that would have irked me! 🤣

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**